src/index.js

```diff
@@ -1,3 +1,33 @@
 import './style.css';
 
-alert('Welcome to webpack setup By JAMES');
\ No newline at end of file
+const form = document.querySelector('form');
+const todoContainer = document.querySelector('.todoContainer');
+
+const lists = [{
+  description: 'Work hard on coding',
+  bool: 'false',
+  index: '1',
+},
+{
+  description: 'Be smart and consistent',
+  bool: 'false',
+  index: '2',
+}, {
+  description: 'You can do it James ',
+  bool: 'false',
+  index: '3',
+}];
```

Comment:
- [ ] Your to-do list is empty! In this activity we want the hard coded task array to be rendered dynamically to the DOM during page load. And the tasks should appear in order of the index values for each task. Kindly ensure to improve upon the same.

**Your to-do list:** 
![image](https://user-images.githubusercontent.com/12745474/193165454-e5d50790-1dc9-4f17-88a9-d40b488f68a1.png)

**Expected:**
![image](https://user-images.githubusercontent.com/12745474/193165925-7de54f89-a581-4de7-b8ec-aa57e2444de5.png)

Reply:
This change has been implemented well.